go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/natefinch/lumberjack v2.0.0+incompatible
 	github.com/op/go-logging v0.0.0-20160315200505-970db520ece7
 	github.com/russross/blackfriday v2.0.0+incompatible
-	github.com/satori/go.uuid v1.2.0
```

Comment:
Please remove this go module.

Reply:
@negiDharmendra It's an indirect module  (used by some other module)

Reply:
My bad